nubilanding/src/components/blog/blog.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import Form from './form';
+
+class Blog extends React.Component {
+  render() {
+    return (
+      <div className={styles.contenedorGeneral}>
```

Comment:
Este componente puede ser definido como función en lugar que de clase ya que no usa state ni ningún método de cliclo de vida, así no se cargan métodos que no se van a usar en el componente.
```suggestion
function Blog() {
  return (
    <div className={styles.contenedorGeneral}>
```